ft232/libftdi.py

```diff
@@ -417,3 +418,6 @@ def setBitMode(self, mask, mode=BM_RESET):
             self.setBitMode(0, self.BM_RESET)
         if ftdi.ftdi_set_bitmode(self._context, mask, mode):
             raise LibFtdiException(self._context)
+
+    def gpio(self):
+        return GPIO(self)
```

Comment:
I don't like this at all. I think the MPSE functionality should just be included in the a single class. For the GPIO code I think you'd probably just want to have a class that LibFtdi and D2XX subclass directly to add the common gpio members.

If user's use this as you have in the example, it ends up creating and destroying a new class every time they want to any GPIO operations....